gammapy/estimators/map/tests/test_ts.py

```diff
@@ -564,12 +564,36 @@ def test_joint_ts_map_hawc():
     datasets = Datasets.read("$GAMMAPY_DATA/hawc/DL4/HAWC_pass4_public_Crab.yaml")
     datasets = Datasets(datasets[-2:])
 
-    estimator = TSMapEstimator(kernel_width=2 * u.deg, sum_over_energy_groups=False)
+    estimator = TSMapEstimator(
+        kernel_width=2 * u.deg, sum_over_energy_groups=False, n_jobs=4
+    )
+    result = estimator.run(datasets)
+    assert_allclose(result["flux"].data[0, 59, 59], 1.909396e-13, rtol=1e-3)
+    assert_allclose(result["sqrt_ts"].data[0, 59, 59], 10.878956, rtol=1e-3)
+
+    estimator = TSMapEstimator(
```

Comment:
Isnt the test getting repeated here?

Reply:
`selection_optional=["stat_scan"]` is added independently in the second case to be sure it returns the same.
I could add a test on the scan output.